Operational-Procedures/Scheduler/night-time-operations/index.rst

```diff
@@ -0,0 +1,255 @@
+
+.. code-block:: text
+
+    component: Scheduler:2
+    cmd: resume
+
+.. figure:: ./_static/atqueue-run-cmd-scheduler-resume.png
+    :name: fig-atqueue-run-cmd-scheduler-resume
+
+    Starting the AT Scheduler from the ATQueue view on LOVE
+
+or from nublado:
+
+.. code:: ipython3
+
+    await remote.cmd_resume.set_start(timeout=5)
+
+Once the Scheduler CSC starts, it will gather talemetry and compute a set of targets to send to the ScriptQueue.
+At this point it is important to keep and eye on the Scheduler logging output, to follow up the different steps it takes during the process.
+
+For the Auxiliary Telescope, the `AT Summary State`_ view provides a good overview of the system.
+A screen shot of this view, taken whith most of the AT components in ``STANDBY`` state is shown :ref:`below <fig-at-summary-state-view>`.
+
+It is also highly recommended to have the `ATQueue`_ view opened at all times.
+The Scheduler will queue 2 observations ahead of time, so you should expect to have one observation executing and another 2 waiting to be executed.
+As the scripts finishes executing, the Scheduler will continue to add targets to the queue.
+
+.. _AT Summary State: http://amor01.cp.lsst.org/uif/view?id=62
+.. _ATQueue: http://amor01.cp.lsst.org/uif/view?id=41
+
+.. figure:: ./_static/at-summary-state-view.png
+    :name: fig-at-summary-state-view
+
+    AT Summary State view from LOVE at the summit.
+    At the top the view contains the summary state of the most relevant components. 
+    At the bottom, on the right-hand side, is shown the dome and telescope position and on the left-hand side the logging from the Scheduler.
+
+.. _scheduler-night-time-operation-troubleshooting:
+
+Troubleshooting
+===============
+
+The following are a collection of documented occurrences and how to resolve them.
+
+.. _scheduler-night-time-operation-troubleshooting-no-target:
+
+No Target
+---------
+
+Multiple factors dictate not only which target the Scheduler will provide next but if *there is a suitable target for observing*.
+
+One the most common occurrences at the beginning of the night is that the Scheduler is initialized early enough that it has no suitable targets for observing.
+In general, that happens because the most common Scheduler configurations only produces valid targets between nautical down and dusk (sun elevation below 12 degrees).
+Nevertheless, this condition may also occurr during the night, if there are gaps in the scheduling configuration.
+In general, users can be aware of these ahead of time by simulating the observing night.
+
+Regardless of when this happen during the night, the user should expect the same response from the Scheduler.
+
+By looking at the log messages commming out from the Scheduler, the user will see the following debug message:
+
+.. code-block:: text
+
+    No target from the scheduler. Stopping with 0.
```

Comment:
Stopping with 0 ?? This is not a very useful error message.

Reply:
It would be useful to say, something like "Stopping Tracking until target is available"

Reply:
yeah, I noticed there's a missing useful log message when this condition occurs. I'll have to update the Scheduler CSC first though.

Reply:
ok.. improved this in the CSC and updated the documentation